src/app/signalen/bedankt/page.tsx

```diff
@@ -1,34 +1,68 @@
+import { useFormContext } from '../FormContext'
+
+function Thanks() {
+  const { updateFormData } = useFormContext()
+
+  useEffect(() => {
+    // Reset form data
+    updateFormData({
+      body: null,
+      when: null,
+      type: null,
+      who: null,
+      phone: null,
+      mail: null,
+      permission: null,
+    })
+  }, [])
 
-function Bedankt() {
   return (
-    <Grid paddingBottom="medium">
-      <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
-        <Column>
-          <Heading>Bedankt!</Heading>
-          <Paragraph>
-            Uw melding is bij ons bekend onder nummer: SIG-14022. Hebt u een e-mailadres ingevuld? Dan ontvangt u een
-            e-mail met alle gegevens van uw melding.
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <Column>
+        <Heading>Melding openbare ruimte</Heading>
+        <Column gap="small">
+          <Heading level={2}>Bedankt</Heading>
+          <Paragraph size="small">
+            Bedankt voor uw melding. Wij hebben uw melding ontvangen op {new Date(Date.now()).toLocaleDateString('nl')}{' '}
+            om {new Date(Date.now()).toLocaleTimeString('nl', { hour: 'numeric', minute: 'numeric' })}. Uw meldnummer is
+            1977916.
           </Paragraph>
 
-          <Heading level={2}>Wat doen we met uw melding?</Heading>
-          <Paragraph>
-            U hoort binnen 3 werkdagen wat we met uw melding hebben gedaan. Dringende meldingen pakken wij zo snel
-            mogelijk op.
-          </Paragraph>
+          <Heading level={3} size="level-4">
+            Wat doen we met uw melding?
+          </Heading>
+          <Paragraph size="small">Meldingen pakken we binnen 3 werkdagen op.</Paragraph>
 
-          <Heading level={2}>Wilt u nog een andere melding doen?</Heading>
+          <Heading level={3} size="level-4">
+            Vragen of meer informatie?{' '}
+          </Heading>
+          <Paragraph size="small">
+            Neem dan contact met ons op via{' '}
+            <ADSLink href="tel:14020" variant="inline">
+              14 020
+            </ADSLink>
+            . Vermeld hierbij alstublieft uw meldnummer. Meer informatie kunt u eventueel ook vinden op{' '}
+            <ADSLink variant="inline" href="https://www.amsterdam.nl/">
+              amsterdam.nl
+            </ADSLink>
+            .
+          </Paragraph>
 
-          <NextLink legacyBehavior passHref href="/signalen">
-            <Link href="/signalen">Doe een melding</Link>
-          </NextLink>
+          <Heading level={3} size="level-4">
+            Wilt u nog een andere melding doen?
+          </Heading>
+          <Link href="/signalen" legacyBehavior passHref>
+            <ADSLink>Doe een melding</ADSLink>
+          </Link>
         </Column>
-      </Grid.Cell>
-    </Grid>
+      </Column>
+    </Grid.Cell>
   )
 }
 
-export default Bedankt
+export default Thanks
```

Comment:
Good point. Do you prefer `Bedankt`? Or simply `Page` for all pages?

Reply:
Not sure, since it's a directory perhaps `index.tsx` with a default export so you can import from `./bedankt`? It's kind of confusing since we use english in code but dutch in the url here.